tests/Argu.Core.Tests/App.config

```diff
@@ -11,6 +11,11 @@
     <assemblyIdentity name="Microsoft.DotNet.PlatformAbstractions" publicKeyToken="adb9793829ddae60" culture="neutral" />
     <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.0.4.0" />
   </dependentAssembly>
+  <dependentAssembly>
+    <Paket>True</Paket>
+    <assemblyIdentity name="Microsoft.Extensions.CommandLineUtils" publicKeyToken="adb9793829ddae60" culture="neutral" />
```

Comment:
These don't hurt, but shouldn't be here. These are `clitool` dependencies. I'll create a paket bug.

Reply:
https://github.com/fsprojects/Paket/issues/2955